src/coreclr/src/vm/fcall.h

```diff
@@ -606,11 +605,13 @@ LPVOID __FCThrowArgument(LPVOID me, enum RuntimeExceptionKind reKind, LPCWSTR ar
         } FORLAZYMACHSTATE_ENDLOOP(alwaysZero);
 
 #define HELPER_METHOD_FRAME_END_EX(gcpoll,allowGC)                          \
+            __helperframe.Pop();                                            \
```

Comment:
Should this be between `UNINSTALL_UNWIND_AND_CONTINUE_HANDLER` and `UNINSTALL_MANAGED_EXCEPTION_DISPATCHER` to match the initialization order?

Reply:
My mistake, of course it should be there.

Reply:
Hmm, moving it there has broken it somehow, I'll take a look in the morning.